wikipedia-sentences/pods/encode.yml

```diff
@@ -0,0 +1,6 @@
+
+!TransformerTorchEncoder
+with:
+  pool_strategy: auto
```

Comment:
auto is deprecated. I would suggest you use `pool_strategy: mean`, and `layer_index=-2`, as the model you are using is not a pretrained classifier model, but a pretrained language model.

Reply:
Ah, although I see you are still using old jina - then ok, you have to wait to update jina for this to work

Reply:
Good to know :+1: I'll bear this in mind when I update it to latest version